source/_integrations/zwave_js.markdown

```diff
@@ -139,11 +168,153 @@ Value Notification example:
+
+- [Z-Wave legacy (1.4) -> ZwaveJS2MQTT](https://community.home-assistant.io/t/switching-from-zwave-1-4-to-zwavejs2mqtt/276721)
+
+You can also visit the #zwave channel in discord for support.
+
+## Frequently Asked Questions
+
+### Can I switch between the Official Z-Wave JS add-on and Z-Wave JS to MQTT?
+
+You can but be aware to not run them both at the same time, only one of them can be active.
+Do remember however that switching requires a re-interview of the network. It is possible to copy the (JSON) cache files but that is out of the scope of this manual. To prevent you from doing all the renaming work again there's a small trick to update the existing Z-Wave JS configuration with the new WebSocket URL: Just re-add the Z-Wave integration to Home Assistant, filling in the new/updated WebSocket URL. There will be a popup raised that this Z-Wave network is already configured but "under the hood" the WebSocket URL is adjusted.
+  
+### I do not see any entities created for my device in Home Assistant
+
+Entities will be created only after the node hits the ready state (its interview is completed). Also, note that some devices (like button remotes) do not create any entities but will only provide events when a button is pressed. See the events section on how to handle those events in your automations. If you are certain that your device should have entities and you do not see them (even after a restart of Home Assistant Core), that will be the time to create an issue about your problem on the GitHub issue tracker, see below section of troubleshooting issues.
+
+### Is there a way to easily export a dump of all my current Z-Wave nodes before I migrate?
+
+You can run the script below in the Developer Tools to get a full oversight of your nodes and their entities.
+
+```yaml
+{%- for node, zstates in states | selectattr('attributes.node_id', 'in', range(1000)) | groupby('attributes.node_id') %}
+
+{%- for s in zstates %}
+Node {{ node }};{{ s.name }};{{ s.entity_id }}{% endfor %}
+
+{%- endfor %}
+```
+
+## How can I add (include) a new device to my Z-Wave network?
+
+1. In Home Assistant, open Settings, Configuration, Z-Wave JS -> Configure.
+2. Press `Add node`.
+3. Press `Start Inclusion`. The Z-Wave controller is now in inclusion mode and will not respond to other commands.
+4. Put the device you want to add in inclusion mode. refer to it's manual how this is done.
+5. The UI should confirm that the node was added and it will be immediately visible in Home Assistant, after a short while (seconds to minutes) the entities should also be created.
+6. If the controller fails to add/find your device, cancel the inclusion process (to unblock your network again). It some cases it might help to first remove a node (exclusion) before you add, even when the device has not been added to this Z-Wave network yet. Another approach would be to factory reset the device. Info about that is in the manual of your device.
```

Comment:
```suggestion
6. If the controller fails to add/find your device, cancel the inclusion process (to unblock your network again). In some cases it might help to first remove a node (exclusion) before you add it, even when the device has not been added to this Z-Wave network yet. Another approach would be to factory reset the device. Info about that is in the manual of your device.
```